packages/@okta/vuepress-site/docs/guides/submit-app-prereq/main/index.md

```diff
@@ -104,7 +104,7 @@ The OIN catalog organizes integrations into use cases. From the OIN Manager, you
 | [Bot or Fraud Detection](https://www.okta.com/integrations/?category=bot-or-fraud-detection) | Provides protection from inauthentic, forged, or otherwise fraudulent attempts to register, sign in, recover, or perform identity-related transactions. Most integrations in this use case are API service integrations that send risk signals to Okta using OAuth 2.0. |
 | [Multifactor Authentication (MFA)](https://www.okta.com/integrations/?category=multi-factor-authentication-mfa) | Provides an extra layer of security with multifactor authentication for an organization's cloud, mobile, and web apps |
 | [Risk Signal Sharing](https://www.okta.com/integrations/?category=risk-signal-sharing)| Provides enriched context on clients, apps, users, and other first-party subjects to augment and inform Okta adaptive authentication and authorization decisions |
-| [Apps for Good](https://www.okta.com/integrations/?category=apps-for-good) | Pre-built integrations with leading apps that enable your workforce to give back. If your organization belongs in the Apps for Good use case, you first need approval from the Apps for Good team. Email [AppsForGood@okta.com](mailto:AppsForGood@okta.com) to request approval.  |
+| [Apps for Good](https://www.okta.com/integrations/?category=apps-for-good) | Pre-built integrations with leading apps that enable your workforce to give back. You can't select this use case. If your organization belongs in the Apps for Good use case, contact the Apps for Good team for approval. Email [AppsForGood@okta.com](mailto:AppsForGood@okta.com).  |
```

Comment:
I recommend just deleting this entire row 107 completely - if its removed from the UI, there's no explainer needed. And the Okta Apps for Good internal team will reach out one-off to any qualifying ISVs so we dont need self service docs for this since denoting it as such will never be self service

Reply:
I agree with removing line 107 - since there is no option to select Apps for Good, we don't need to explain what it means. Agree with Albert's reasonings